src/serverdlg.h

```diff
@@ -91,8 +92,14 @@ public slots:
     void OnLocationCountryActivated ( int iCntryListItem );
     void OnCentServAddrTypeActivated ( int iTypeIdx );
     void OnTimer();
+
     void OnServerStarted() { UpdateSystemTrayIcon ( true ); }
-    void OnServerStopped() { UpdateSystemTrayIcon ( false ); }
+    void OnServerStopped()
+    {
+         UpdateSystemTrayIcon ( false );
```

Comment:
Oh, I see you have changed it. I should have been more precise in my initial post. I only prefer to put it in the header file if it is a single command. Just in some very special cases I put more than one in the header. But in this case, it should belong to the cpp file as you had it in your original commit. Sorry for the confusion...

Reply:
No problem -- Both initially changed.  Then I changed Started not to change.  But left it in the cpp.  I'll split them up :).